src/infrastructure/database/migrations/versions/2024_06_17_10_56-merge_multiinformant_migrations.py

```diff
@@ -0,0 +1,21 @@
+"""Merge multiinformant migrations
+
+Revision ID: affe09e93102
+Revises: 297c9d675e2f, f436ffa84be1
```

Comment:
Looks like this was updated to revise `2024_06_21_11_24-flow_assessments.py`. Could this cause the flow_assessments migration to be skipped where Alembic already performed Revision `affe09e93102`

Reply:
Yes, but I did this to work on envs that already have 2024_06_21_11_24-flow_assessments.py applied, like production. without this change it may not work.

To make it work on environments that already have multiinformant, there is a `hack`

- Comment out content of `downgrade` function on 2024_06_21_11_24-flow_assessments.py file, add `pass` as content
- run `alembic downgrade c587d336f28e`
- Undo changes in 2024_06_21_11_24-flow_assessments.py 
- run `alembic upgrade head`

This should be done in your local environment, dev3 and stage environment. In UAT and production it will work properly

Reply:
I was able to downgrade then upgrade to head again after commenting it out as suggested. Remaining queries now work with the migrations successfully applied. 

Agreed it won't be needed on UAT/Prod.